src/components/dashboard/appShell/header.tsx

```diff
@@ -3,122 +3,103 @@ import {
   Box,
   BoxProps,
   Flex,
-  Heading,
   Image,
-  Modal,
-  Text,
+  Portal,
+  useBoolean,
   useColorModeValue,
 } from '@chakra-ui/react';
-import React, { useEffect, useState } from 'react';
-import { MobileMenu } from './mobileMenu';
+import React, { FC, useEffect, useRef, useState } from 'react';
 import { Link } from 'react-router-dom';
-import VaccinationPass from '../../../assets/VaccinationPass.png';
+import VaccinationPass from '../../../assets/VaccinationPassV2.png';
 import { useKeycloak } from '@react-keycloak/web';
 import { KeycloakProfile } from 'keycloak-js';
+import { HamburgerIcon } from '@chakra-ui/icons';
+import { IconType } from 'react-icons';
+import { FaBookMedical, FaRegCalendarAlt, FaSyringe } from 'react-icons/fa';
+import { NavItem } from './navitem';
 
 interface DashboardProps extends BoxProps {
   onOpen: () => void;
   onClose: () => void;
   isOpen: boolean;
 }
 
-export const DashboardHeader = ({
-  onClose,
-  onOpen,
-  isOpen,
-  ...rest
-}: DashboardProps) => {
+interface LinkItemProps {
+  name: string;
+  icon: IconType;
+  link: string;
+}
+
+const LinkItems: Array<LinkItemProps> = [
+  { name: 'History', icon: FaSyringe, link: 'history' },
+  { name: 'Recommendations', icon: FaRegCalendarAlt, link: 'recommendations' },
+  { name: 'Wiki', icon: FaBookMedical, link: 'wiki' },
+];
+
+export const DashboardHeader: FC<DashboardProps> = ({}) => {
   const { keycloak } = useKeycloak();
   const [profile, setProfile] = useState<KeycloakProfile | undefined>();
```

Comment:
```suggestion
  // eslint-disable-next-line no-unused-vars
  const [profile, setProfile] = useState<KeycloakProfile | undefined>();
```
@FuchsDominik use this if you want to keep this code around for later use.

Reply:
Good idea, this way we solve the linting error